app/operations/mailers/user_added_to_project.rb

```diff
@@ -0,0 +1,10 @@
+module Mailers
+  class UserAddedToProject < Operation
+    integer :resource_id
+    array :roles
+
+    def execute
+      UserAddedToProjectMailerWorker.perform_async(api_user.id, resource_id, roles)
```

Comment:
Looking how this turned out i agree with marten but it's not a blocker for me. Just another layer of abstraction that doesn't really do anything since there is no logic in this particular operation.